src/simplicial-hott/08-covariant.rzk.md

```diff
@@ -2228,3 +2227,52 @@ commuting with the contravariant lifts.
              x y f (g b))))
 
 ```
+
+## Discrete fibers
+
+The fibers of a covariant fibration over a Segal type are discrete types.
+
+```rzk title="RS17, Proposition 8.18"
+#def is-discrete-is-covariant-segal
+  ( A : U)
+  ( is-segal-A : is-segal A)
+  ( C : A → U)
+  ( is-cov-C : is-covariant A C)
+  ( x : A)
+  : is-discrete (C x)
+  :=
+    ( \ u v → total-equiv-family-of-equiv
```

Comment:
I'd start a new line after the -> and align the "total-equiv-family-of-equiv" with the "\" of "\ u v". This will make it clearer that the things to follow are arguments of the function "total-equiv-family-of-equiv".